tasks/mrrc-collect-params/mrrc-collect-params.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: mrrc-collect-params
+spec:
+  params:
+    - name: dataJsonPath
+      description: path to data json file
+  results:
+    - name: mrrcParamFilePath
+      description: path of the env file for following tasks
+    - name: cosignPubKeyFile
+      description: the public key file name for cosign verification
+    - name: cosignPubKeySecret
+      description: the secret name for cosign verify public key
+    - name: charonAWSSecret
+      description: the secret name for charon aws credential file
+  steps:
+    - name: mrrc-collect-params
+      image: quay.io/konflux-ci/yq:latest
+      script: |
+        set -eux
+
+        cd $(workspaces.data.path)
+        mrrcEnvFile="./mrrc.env"
+        if [ -f $mrrcEnvFile ]; then
+          rm $mrrcEnvFile
+        fi
+
+        zipRegistry=$(yq -r '.zipRegistry' $(params.dataJsonPath))
```

Comment:
Any reason to use `yq` instead of `jq`? I understand that it will likely work, but I'd default to `jq` when working with json files - if nothing else, it makes us consistent.

Reply:
This is because the task is using quay.io/konflux-ci/yq:latest which just provided yq. I don't know what's the official image from konflux which can provide jq.

Reply:
Oh I see, I didn't notice that. What do you think about using our release-service-utils image? It's used in the vast majority of tasks and we plan to update it automatically in the future. It's defined here: https://github.com/konflux-ci/release-service-utils/blob/main/Dockerfile

Reply:
Fixed in new commit